Gemfile

```diff
@@ -1,7 +0,0 @@
-source 'https://rubygems.org'
```

Comment:
Why did you delete this? I understand that you needed this for mac... but we can't build without this,